### ISSUE TITLE
Change the IFF for all tankers to make it more consistent with C/S

### ISSUE THREAD

**Issue by Heidana** [132nd-vWing/ATRM#157]
**Is your feature request related to a problem? Please describe.**
Currently the IFF for thankers are set via the unit and pilot name e.g. "ARCO AR 203 #IFF:5322FR"
AR402 and 403 are using invalid codes 5318 & 5319 respectively.

**Describe the solution you'd like**
My proposal is to change the IFFs so it corresponds to the Tanker C/S:
AR101 ->5101FR
AR20X -> 520XFR
AR30X -> 530XFR
AR40X -> 640XFR

**Describe alternatives you've considered**
Alternatively only the 2 invalid codes can be changed to something else 5322 and 5323 for example.

**Additional context**
This will help the F-15E to interrogate and find the correct tankers much easier.
I can help with updating the ATRM brief if approved.

**Comment by 132ndNeck** [comment]
As of now work on the ATRM is on the backburner in anticipation of the building of a new training mission on the Kola map when it drops. Because of this, I will not redo all IFF codes, but I have changed the IFF codes for the invalid codes of the two tankers AR402 and AR403

New codes: 
AR402: 5322
AR403: 5323

**Comment by 132ndNeck** [comment]
@132nd-Entropy 
When I changed the names per this request, I did not update it in the script(s). It has been reported on a couple events now that the tankers are not spawned,  and I just now realized this is because I made this change in the name and not the script.

Are you  able to help out with updating the name in the correct scripts? (I am not quite sure which/where to make the necessary change)
